pkg/antctl/command_definition.go

```diff
@@ -133,7 +143,7 @@ func (cd *commandDefinition) applySubCommandToRoot(root *cobra.Command, client *
 	}
 	renderDescription(cmd, isAgent)
 
-	cd.applyFlagsToCommand(cmd)
+	cd.applyFlagsToCommand(cmd, cd.Flags)
```

Comment:
I think the `cd.Flags` parameter is not needed.

Reply:
Thank you, it is removed now.

Added validation for:
* no empty flagInfo name
* flagInfo name cannot be the same
* length of flagInfo shorthand cannot be larger than 1